docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -354,6 +354,38 @@ public class SomeTest {
 ----
 ====
 
+[[hr-named-datasource]]
+==== Named data sources
+
+Hibernate Reactive supports having named data sources
+
+[source,properties]
+.Example `{config-file}`
+----
+# datasource configuration
+quarkus.hibernate-orm.datasource = named-datasource
+quarkus.datasource."named-datasource".reactive", true
+quarkus.datasource."named-datasource".db-kind", postgresql
+quarkus.datasource."named-datasource".username", quarkus_test
+quarkus.datasource."named-datasource".password", quarkus_test
```

Comment:
Copy-pasted from tests, I assume?

```suggestion
quarkus.datasource."named-datasource".reactive = true
quarkus.datasource."named-datasource".db-kind = postgresql
quarkus.datasource."named-datasource".username = quarkus_test
quarkus.datasource."named-datasource".password = quarkus_test
```

Reply:
Though really, the minimal configuration is just this:

```suggestion
quarkus.datasource."named-datasource".db-kind = postgresql
```

The rest is just the defaults and/or would need to be different in prod anyway.